src/main/java/com/dream11/aerospike/factory/AerospikeClientFactoryImpl.java

```diff
@@ -0,0 +1,49 @@
+package com.dream11.aerospike.factory;
+
+import com.aerospike.client.Host;
+import com.aerospike.client.policy.ClientPolicy;
+import com.dream11.aerospike.client.AerospikeClient;
+import com.dream11.aerospike.client.AerospikeClientImpl;
+import com.dream11.aerospike.config.AerospikeConnectOptions;
+import com.dream11.aerospike.util.PolicyUtil;
+import com.dream11.aerospike.util.SharedDataUtils;
+import io.vertx.core.*;
+import lombok.extern.slf4j.Slf4j;
+
+
+@Slf4j
+public class AerospikeClientFactoryImpl implements AerospikeClientFactory {
+
+  @Override
+  public AerospikeClient getClient(Vertx vertx) {
+    AerospikeConnectOptions config = new AerospikeConnectOptions();
+    return getClient(vertx, config);
+  }
+
+  @Override
+  public AerospikeClient getClient(Vertx vertx, AerospikeConnectOptions config) {
+    ClientPolicy clientPolicy = PolicyUtil.getClientPolicy();
+    return getClient(vertx, clientPolicy, config);
+  }
+
+  @Override
+  public AerospikeClient getClient(Vertx vertx, ClientPolicy clientPolicy, AerospikeConnectOptions config) {
```

Comment:
instead of using both `ClientPolicy` and `AerospikeConnectOptions`, can we use only `AerospikeConnectOptions` by making `AerospikeConnectOptions extends ClientPolicy`